scss/_card.scss

```
@@ -104,6 +104,11 @@
   margin-bottom: -$card-spacer-y;
   margin-left: -$card-spacer-x / 2;
   border-bottom: 0;
+
+  .nav-link.active {
+    background: $card-bg;
```

Don’t use the shorthand, use `background-color` so it doesn’t override additional `background-` properties.

```suggestion
    background-color: $card-bg;
```